starters/adoc/modules/starters/pages/simpleapp.adoc

```diff
@@ -610,9 +613,6 @@ This may be required because the application might otherwise be configured to us
 
 <.> Sets up logging to use the configuration defined in the `log4j2-test.xml` file
 
-<.> Tests typically inherit from `IsisIntegrationTestAbstract`, which provides some convenience methods to inherit from.
-In this case, the test inherits from the `IsisIntegrationTestAbstractWithFixtures` subclass which also adds in support for running fixtures.
-
 
 
 === webapp's src/main/java
```

Comment:
Beyond a couple of typos, I rearranged the numbered annotations for `SimpleModuleIntegTestAbstract.java` in this file as I think they weren't lined up properly with the line of code they were supposed to correspond to. I made some educated guesses and assumptions here so it is possible that these might not be completely correct

Reply:
looks good to me - thanks!